dotcom-rendering/src/lib/ad-targeting.ts

```diff
@@ -1,27 +1,37 @@
-export const buildAdTargeting = (
-	CAPI:
-		| CAPIType
-		| CAPIBrowserType
-		| Pick<CAPIType, 'isAdFreeUser' | 'config'>,
-): AdTargeting => {
-	if (CAPI.isAdFreeUser) {
+export const buildAdTargeting = ({
+	isAdFreeUser,
+	isSensitive,
+	edition,
+	section,
+	sharedAdTargeting,
+	adUnit,
+	videoDuration,
+}: {
+	isAdFreeUser: boolean;
+	isSensitive: boolean;
+	edition: string;
+	section: string;
+	sharedAdTargeting: Record<string, unknown>;
+	adUnit: string;
+	videoDuration?: number;
+}): AdTargeting => {
+	if (isAdFreeUser) {
 		return {
 			disableAds: true,
 		};
 	}
-	const { config } = CAPI;
 	const customParams = {
-		sens: config.isSensitive ? 't' : 'f',
+		sens: isSensitive ? 't' : 'f',
 		si: 'f',
-		vl: config.videoDuration || 0,
-		cc: config.edition,
-		s: config.section,
+		vl: videoDuration || 0,
+		cc: edition,
+		s: section,
 		inskin: 'f',
-		...config.sharedAdTargeting,
+		...sharedAdTargeting,
 		pa: 'f',
```

Comment:
Looking it this makes me raise a couple of eyebrows, but I appreciate that this PR doesn’t really change the functionality of it all.

Note to self: Are we saying the user is never signed in (`si`)? No personalised advertising (`pa`) ever? Video length (`vl`) should always ever be a multiple of 30… and a string.

Reply:
Signed in is false (and I assume other defaults) because of SSR.
We can't determine that server-side without cache-splitting.
For `YoutubeAtom` `si` gets overridden client-side.